icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2041,6 +2046,49 @@ class U_I18N_API NumberFormatterSettings {
      */
     Derived scale(const Scale &scale) &&;
 
+    /**
+     * Specifies the usage of the unit ("person", "road", "person" ...etc.)
+     *
+     * <p>
+     * NOTE: `usage` will change the output unit depending on the `Locale`  
+     * and the unit value.
+     *    For Example:
+     *        Locale: en_US
+     *        Usage : length-person
```

Comment:
The usage should be "person", not "length-person".

The fact that it is a length is determined by the input measure. The category="xxx" is purely internal, and may disappear.

Reply:
Documentation doesn't have the category prefix anymore.